resource/src/main/resources/application.properties

```diff
@@ -1,8 +1,19 @@
-aws.s3.endpoint-url=http://localhost:4566
```

Comment:
Use YAML.

Reply:
Will change it with the next task's implementation